spotinst/resource_spotinst_administration_org_policy.go

```diff
@@ -0,0 +1,144 @@
+package spotinst
```

Comment:
1. Remove "administration" from resource names.
2. Move the resource outside commons.